lib/requestability_resolver.js

```diff
@@ -26,6 +26,9 @@ class RequestabilityResolver {
             physRequestableCriteria = `${deliveryInfo.deliveryLocation &&
               deliveryInfo.deliveryLocation.length || 0} delivery locations.`
           }
+          if (isInSchomburg(item)) {
+            deliveryInfo = getSchomburgDeliveryInfo(item, deliveryInfo)
+          }
```

Comment:
All of this work is temporary and will be removed for batch 2?

Reply:
yup! unless there is some unforeseen exception (which i think is at least moderately likely).